### PR TITLE
Fix shebang and improve code style

### LINKED ISSUE #12: Improve styling; Conforming to pycodestyle standards
I have worked on the file `SortDisplay.py`, and have implemented all changes except **E203 whitespace before ':'**
I think line# 107-124 look more readable without the E203 fix.

### PR DESCRIPTION
**Pull request by vharsh**
Fixes #12 
Changed 
* shebang
* Code now confirms to Pycodestyle

TODO
* Improve coding style if other Python files

### CODE REVIEW

File: SortDisplay.py [hunk 1]
```diff
@@ -20,8 +21,7 @@ def __init__(self, algorithm, stop_event, options, compare=None):
         self.compare = compare
 
         self.stop = False
-        self.cstop = compare == None
-
+        self.cstop = True if compare is None else False
```
Comment (Eckankar) [inline SortDisplay.py]: I prefer the shorter form: `compare is None`

File: SortDisplay.py [hunk 2]
```diff
@@ -40,10 +40,12 @@ def __init__(self, algorithm, stop_event, options, compare=None):
 
         pygame.init()
 
-        if compare == None:
-            self.window = pygame.display.set_mode((self.width, 6 * (self.numLines+1)))
+        if compare is None:
+            self.window = pygame.display.set_mode((self.width,
+                                                   6 * (self.numLines+1)))
```
Comment (Eckankar) [inline SortDisplay.py]: I don't think this line needs to be wrapped.

File: SortDisplay.py [hunk 3]
```diff
@@ -40,10 +40,12 @@ def __init__(self, algorithm, stop_event, options, compare=None):
...
         else:
-            self.window = pygame.display.set_mode((self.width * 2 + 5, 6 * (self.numLines+1)))
+            self.window = pygame.display.set_mode((self.width * 2 + 5,
+                                                   6 * (self.numLines+1)))
```
Comment (Eckankar) [inline SortDisplay.py]: Nor this one.

File: SortDisplay.py [hunk 4]
```diff
@@ -54,15 +56,16 @@ def update(self):
 
         self.__update_algorithm(self.items.items, self.markers.markers, 0)
 
-        if self.compare != None:
-            self.__update_algorithm(self.citems.items, self.cmarkers.markers, self.width + 5)
+        if self.compare is not None:
+            self.__update_algorithm(self.citems.items,
+                                    self.cmarkers.markers, self.width + 5)
```
Comment (Eckankar) [inline SortDisplay.py]: Nor this one.

File: SortDisplay.py (hunk 1 above)
Comment (vharsh) [inline SortDisplay.py]: Shall I revert it back to it's old version? I can't find another way to accomplish this in one line.

**Comment by vharsh** [comment]
I am a bit confused, I did lots of git reset to compress the commits into one logical commit. I don't know if I can refer it here. 
Thanks for bearing with me. :smile:  :+1:

**Comment by Eckankar** [comment]
@vharsh No worries. I'll merge it in.

**Comment by Eckankar** [comment]
Merged in as https://github.com/Eckankar/PySortDemo/commit/7c25c8cf5133c07e0fbe676b1a0eb35d7379b839

Thank you.

**Pull request closed without merging**